content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/title.html

```diff
@@ -17,6 +17,6 @@
      data-sly-use.title="com.adobe.cq.wcm.core.components.models.Title"
      data-sly-use.template="core/wcm/components/commons/v1/templates.html"
      data-sly-test.text="${title.text}">
-    <h1 class="cmp-title__text" data-sly-element="${title.type}">${text}</h1>
+    <h1 id = "${title.id}" class="cmp-title__text" data-sly-element="${title.type}">${text}</h1>
```

Comment:
Please remove the space around the = sign to keep the style of the rest of the file. In HTML, unlike other programming languages, it would be uncommon to have spaces around the = signs.